src/pages/PrintPage.tsx

```diff
@@ -233,12 +239,14 @@ class SummaryPage extends React.Component<RouteComponentProps, State> {
         <Modal
           isOpen={this.state.showConfirmModal}
           centerContent
+          ariaLabel=""
           content={
-            <Prose>
+            <Prose id="modalaudiofocus">
               <Text center>
                 You may not make any changes after you print your ballot.
               </Text>
               <Text center>Do you want to print your ballot?</Text>
+              <span aria-label="Use the down arrow to continue." />
```

Comment:
I think this much safer/future-proof solution than ending aria-label's with "then".

Reply:
It doesn't really work as well for the other flows. I promise, I've tried a lot of things :)